src/serializer/visitors.js

```diff
@@ -116,6 +135,22 @@ export let ClosureRefReplacer = {
     }
   },
 
+  // TODO: handle FunctionDeclaration
+  FunctionExpression(path: BabelTraversePath, state: ClosureRefReplacerState) {
+    if (t.isProgram((path: any).parentPath.parentPath.node)) {
+      // Skip itself.
```

Comment:
I don't understand why this is true in the general case or why it is safe to do this dereference.

Reply:
Because the code we are traverse is always wrapped into file AST via:
```js
t.file(t.program([t.expressionStatement(funcNode)
```
So the path.parentPath.parentPath will be safe to dereference. I will add a comment though.